doc/classes/InputEventJoypadButton.xml

```diff
@@ -49,13 +50,13 @@
 	</methods>
 	<members>
 		<member name="button_index" type="int" setter="set_button_index" getter="get_button_index">
-			Joypad button identifier, one of the JOY_BUTTON_* constants in [@Global Scope].
+			Button's identifier, one of the JOY_BUTTON_* constants in [@Global Scope].
```

Comment:
The button's identifier. One of the [code]JOY_BUTTON_*[/code] constants from [@Global Scope].